src/client.rs

```diff
@@ -83,6 +83,7 @@ impl GrpcClient {
             Header::new(":path", method.name.clone()),
             Header::new(":authority", host.clone()),
             Header::new(":scheme", http_scheme.as_bytes()),
+            Header::new("content-type", "application/grpc"),
```

Comment:
Not sure if it will support json later, btw the golang implementation is also always use "application/grpc".